api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -509,6 +480,13 @@ public void validate_OperationalUser_nullInstitution() {
     assertThat(service.validateOperationalUser(institution)).isFalse();
   }
 
+  @Test(expected = BadRequestException.class)
```

Comment:
good test - but let's add all the other fields to make it more clear which one is the offender.